src/client/eppo-client.ts

```diff
@@ -726,9 +755,9 @@ export default class EppoClient {
     try {
       if (this.banditLogger) {
         this.banditLogger.logBanditAction(banditEvent);
-      } else if (this.queuedBanditEvents.length < MAX_EVENT_QUEUE_SIZE) {
+      } else {
         // If no logger defined, queue up the events (up to a max) to flush if a logger is later defined
-        this.queuedBanditEvents.push(banditEvent);
+        this.banditEventsQueue.push(banditEvent);
```

Comment:
no limit anymore ya?

Reply:
the limit is now encapsulated inside the logic of `BoundedEventQueue#push` method!